users/vieting/experiments/switchboard/ctc/feat/network_helpers/specaug_stft.py

```diff
@@ -0,0 +1,163 @@
+from typing import Dict, Union, Optional, List
+from i6_core.returnn.config import CodeWrapper
+from .rc_specaug import _mask_v1, random_mask_v1, specaugment_v1_eval_func
+
+
+def add_specaug_layer(
+    network: Dict,
+    name: str = "specaug",
+    from_list: Optional[Union[str, List[str]]] = None,
+    max_time_num: int = 3,
+    max_time: int = 10,
+    max_feature_num: int = 4,
+    max_feature: int = 5,
+) -> List[str]:
+    if from_list is None:
+        from_list = ["stft"]
```

Comment:
I would keep data here, that is more generic. For your case of STFT-domain experiments, you always overwrite `from_list` anyway.
```suggestion
        from_list = ["data"]
```